js/main.js

```diff
@@ -0,0 +1,87 @@
+const getRandomPositiveInteger = (a, b) => {
+    const lower = Math.ceil(Math.min(Math.abs(a), Math.abs(b)));
+    const upper = Math.floor(Math.max(Math.abs(a), Math.abs(b)));
+    const result = Math.random() * (upper - lower + 1) + lower;
+    return Math.floor(result);
+};
+const getRandomArrayElement = (elements) => {
+    return elements[getRandomPositiveInteger(0, elements.length - 1)];
+};
+const AVATAR_IMG = [
+    'img/avatar-1','img/avatar-2','img/avatar-3','img/avatar-4','img/avatar-5','img/avatar-6'
+];
+const COMMENTER_NAMES = [
+    'Имя1','Имя2','Имя3','Имя4','Имя5','Имя6','Имя7','Имя8','Имя9','Имя10'
+];
+const COMMENT_MESSAGES = [
+    'Всё отлично!',
+    'В целом всё неплохо. Но не всё.',
+    'Когда вы делаете фотографию, хорошо бы убирать палец из кадра. В конце концов это просто непрофессионально.',
+    'Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.',
+    'Я поскользнулся на банановой кожуре и уронил фотоаппарат на кота и у меня получилась фотография лучше.',
+    'Лица у людей на фотке перекошены, как будто их избивают. Как можно было поймать такой неудачный момент?!'
+];
+
+let photo_urls = [];
+let photo_descriptions = [];
+let ids_for_comments = [];

```

Comment:
В целом, это не так важно, но по критериям, обычные перменные называются в camelCase Нотации, т.е. без нижних подчерикваний, на их работу это никак не сказывается, просто условность, постарайся придерживаться на будущее.
```suggestion
let photoUrls = [];
let photoDescriptions = [];
let idsForComments = [];
```